ui/src/components/imquery/ConceptSelect.vue

```diff
@@ -17,26 +17,20 @@
     </div>
     <div class="value-list-container" v-if="isValueList">
       <div class="value-list" v-if="isArrayHasLength(values)">
-        <div v-if="isType === 'Entity'" class="value-list-item" v-for="[index, value] of values.entries()">
+        <div class="value-list-item" v-for="[index, value] of values.entries()">
           <EntailmentOptionsSelect :entailment-object="value" @update-entailment="onUpdateEntailment" />
-          <SelectButton v-model="isType" :options="['Entity', 'Cluster code']" @change="handleIsTypeChange(value)" />
+          <SelectButton v-model="isType" :options="['Concept', 'Concept set']" @change="handleIsTypeChange(value)" />
           <AutocompleteSearchBar
             :quick-type-filters-allowed="[IM.CONCEPT, IM.CONCEPT_SET]"
             :selected-quick-type-filter="quickTypeFilter"
             :im-query="imQuery"
-            @update-selected-filters="onUpdateSelectedFilters"
-            :selected="{ iri: value['@id'], name: value.name } as SearchResultSummary"
             :root-entities="[datatype]"
+            :selected="{ iri: value['@id'], name: value.name } as SearchResultSummary"
```

Comment:
Casting here will likely result in a double search. Get in correct type from entityService.getEntitySummary.